src/stories/components/Forms/Select/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+import { SelectChangeEvent } from '@mui/material/Select';
+import { Option } from 'stories/components/Forms/Select/types/Option';
+import { Props } from 'stories/components/Forms/Select/types/Props';
+
+import * as S from './styles';
+
+export const Select: React.FC<Props> = ({ options, helperText, ...props }) => {
+  const [value, setValue] = useState<Option['value']>('');
+
+  function onChange(event: SelectChangeEvent<Option['value']>) {
```

Comment:
Sugiro receber uma função `onChange` como prop para quando alguém for utilizar esse componente consiga pegar as mudanças. Deixando a função `onChange` somente dentro do componente quem for usar não vai conseguir ver as mudanças